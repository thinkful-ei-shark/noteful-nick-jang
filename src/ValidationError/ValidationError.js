import React from 'react';
import PropTypes from 'prop-types';
import './ValidationError.css';

export default function ValidationError(props) {
  if (props.message)
    return <p id={props.id} className='validation-error'>{props.message}</p>;
  return null;
}

ValidationError.defaultProps = {
  message: ''
}

ValidationError.propTypes = {
  id: PropTypes.string,
  message: PropTypes.string
}