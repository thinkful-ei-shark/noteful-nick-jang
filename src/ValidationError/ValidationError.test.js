import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import ValidationError from './ValidationError'

describe(`ValidationError component`, () => {
  it('renders ValidationError by default', () => {
    const wrapper = shallow(<ValidationError />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
