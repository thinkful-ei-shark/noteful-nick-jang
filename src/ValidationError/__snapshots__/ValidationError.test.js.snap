// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ValidationError component renders ValidationError by default 1`] = `""`;
